DigitalLearningSolutions.Web/Controllers/LoginController.cs

```diff
@@ -34,14 +34,14 @@ ILogger<LoginController> logger
             this.logger = logger;
         }
 
-        public IActionResult Index(string? returnUrl = null)
+        public IActionResult Index(string? returnUrl = null, string? queryString = null)
         {
             if (User.Identity.IsAuthenticated)
             {
                 return RedirectToAction("Index", "Home");
             }
 
-            var model = new LoginViewModel(returnUrl);
+            var model = new LoginViewModel(returnUrl, queryString);

```

Comment:
Is there a reason not to stick `returnUrl` and `queryString` together here, rather than passing them separately through TempData?

Reply:
Thinking about it, none that I can think of. I've combined them and tested that it still works.